src/components/Details/Details.tsx

```diff
@@ -1,7 +1,67 @@
-import React from 'react';
+import React from "react";
+import "../Details/Details.css"
 
-interface DetailsProps {}
+interface MarketProps {
+  markets: {
+    id: number;
+    distanceFromZip: number;
+    marketName: string
+  }[],
+  marketDetails: {
+    street: string,
+    city: string,
+    state: string,
+    zip: string,
+    schedule: {
+      dayOfWeek: string,
+      time: string,
+      season: string
+    }[],
+    products: string[],
+    mapsLink: string
+  }[],
+  id: string;
+}
+
+const Details: React.FC<MarketProps> = ({ id, markets, marketDetails }) => {
+
+  const marketMatch = markets.filter(market => market.id === Number(id));
+  const detailsMatch = marketDetails.filter(market => { 
+    let match = market.mapsLink.split('%22')[1].includes(marketMatch[0].marketName.split(' ')[0]);
+    console.log(marketMatch[0].marketName)
+    return match
```

Comment:
This _could_ be a direct return of line 30. No need to let variable "match", but it works!

Reply:
oh yeah this is changed too